### PR TITLE
Add Kindergarten Garden exercise

### PR DESCRIPTION
**Pull request by Steffan153**

### CODE REVIEW

File: exercises/practice/kindergarten-garden/kindergarten-garden_test.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
+-- Setup test table and read in student solution:
+.read ./test_setup.sql
+
+-- Test cases:
+INSERT INTO tests (name, uuid, diagram, student, expected)
+    VALUES
+        ('garden with single student', '1fc316ed-17ab-4fba-88ef-3ae78296b692', 'RC
+GG', 'Alice', 'radishes,clover,grass,grass'),
```
Comment (IsaacG) [inline exercises/practice/kindergarten-garden/kindergarten-garden_test.sql:8]:
```suggestion
        ('garden with single student', '1fc316ed-17ab-4fba-88ef-3ae78296b692', 'RC' || char(10) || 'GG', 'Alice', 'radishes,clover,grass,grass'),
```

Thoughts?

Comment (BNAndras) [reply exercises/practice/kindergarten-garden/kindergarten-garden_test.sql:8]: Nice. I like the explictness

Comment (Steffan153) [reply exercises/practice/kindergarten-garden/kindergarten-garden_test.sql:8]: Looks good. Will fix later when I'm on my computer

**Review by IsaacG — APPROVED** [review]

**Review by BNAndras — APPROVED** [review]

**Comment by BNAndras** [comment]
Merge conflict needs to be resolved.

**Review by IsaacG — APPROVED** [review]

**Pull request merged**